### DISCUSSION TITLE
install1804 fails to install nvm

### DISCUSSION THREAD

**Discussion by junukwon7** [hydro-dev/Hydro#43] [General]
install1804 fails to install nvm. Tested in multipass ubuntu18.04(Hyper-V) twice.
The script below didn't install nvm. With no output.
However, it worked after I install it manually.
```
echo "Installing NodeJS"
curl -o- https://cdn.jsdelivr.net/gh/nvm-sh/nvm@v0.36.0/install.sh | sed 's/raw.githubusercontent.com\/nvm-sh\/nvm\//cdn.jsdelivr.net\/gh\/nvm-sh\/nvm@/g' | bash
export NVM_DIR=/root/.nvm
[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
nvm install 14
```

**Comment by undefined-moe** [comment]
please refer to [https://github.com/nvm-sh/nvm/blob/master/README.md](https://github.com/nvm-sh/nvm/blob/master/README.md) .
It's probably not a problem of hydro. please confirm you are running the script as root and nvm is installed to /root/.nvm (at least curl should output something)
